src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -3154,6 +3225,23 @@ private Message GetSortedSetLengthMessage(RedisKey key, double min, double max,
             return Message.Create(Database, flags, RedisCommand.ZCOUNT, key, from, to);
         }
 
+        private Message GetSortedSetIntersectionLengthMessage(RedisKey[] keys, long limit, CommandFlags flags)
+        {
+            if (keys == null) throw new ArgumentNullException(nameof(keys));
+
+            List<RedisValue>? values = new List<RedisValue>();
```

Comment:
No need for nullable here, and we can avoid `List<T>` resizes. For example:
```cs
var values = new List<RedisValue>(1 + keys.Length + (limit > 0 ? 2 : 0));
```
This will construct a list of the correct size from the onset and avoid copies with more than a few keys in play.